src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_vm_diagnostics_templates.py

```diff
@@ -278,248 +278,702 @@ def get_default_diag_config(is_windows):
     else:
         return {
             "ladCfg": {
```

Comment:
This is a data file in nutshell, to get CI passing, it is fine to just disable the flake8 errors by putting `# flake8: noqa` at the top. This way we can just copy and paste new config moving forward